neuralet-distancing.py

```diff
@@ -18,9 +18,9 @@ def start_engine(config, video_path):
 
 
 def start_web_gui(config):
```

Comment:
I suggest renaming this too, to something indicating that it starts the processor's api